hazelcast/src/test/java/com/hazelcast/map/impl/MapStatisticsAwareServiceTest.java

```diff
@@ -39,11 +39,11 @@
 public class MapStatisticsAwareServiceTest extends HazelcastTestSupport {
 
     @Test
-    public void getStats() {
+    public void getStats_whenMapHasData() {
         HazelcastInstance hz = createHazelcastInstance();
         warmUpPartitions(hz);
         // when one map exists
-        hz.getMap("map");
+        hz.getMap("map").put(1, 1);
         MapService mapService = getNodeEngineImpl(hz).getService(MapService.SERVICE_NAME);
         Map<String, LocalMapStats> mapStats = mapService.getStats();
 
```

Comment:
Add another test for members who have a `MapContainer` but no `MapProxy`? something like:
```
    @Test
    public void getStats_whenMapProxyDoesNotExist() {
        HazelcastInstance hz1 = factory.newHazelcastInstance();
        hz1.getMap("map").put(1,1);
        // on the second member the map container will be created but no map proxy
        HazelcastInstance hz2 = factory.newHazelcastInstance();
        waitAllForSafeState();

        MapService mapService = getNodeEngineImpl(hz2).getService(MapService.SERVICE_NAME);
        Map<String, LocalMapStats> mapStats = mapService.getStats();

        // then we obtain 1 local map stats instance
        assertEquals(1, mapStats.size());
        assertNotNull(mapStats.get("map"));
    }
```